doc/man7/property.pod

```diff
@@ -52,15 +52,15 @@ The full syntax for property definitions appears below.
 
 Each implementation of an algorithm can define any number of
 properties.
-For example, the default provider defines the property I<default=yes>
+For example, the default provider defines the property I<provider=default>
 for all of its algorithms.
-Likewise, the FIPS provider defines I<fips=yes> and the legacy provider
-defines I<legacy=yes> for all of their algorithms.
+Likewise, the FIPS provider defines I<provider=fips> and the legacy provider
```

Comment:
Is there only one FIPS provider?  Should that be "a FIPS provider"?  And didn't @slontis say that a FIPS-validated provider could have non-FIPS algorithms in it?

Reply:
It should probably say "OpenSSL's FIPS provider"

Reply:
I'm okay leaving it as is or using @levitte's suggestion.

The FIPS provider can and I suspect will have non-validated algorithms, it is still the FIPS provider.

Reply:
I fixed this using @levitte's suuggestion